src/main/java/au/com/codeka/carrot/Bindings.java

```diff
@@ -0,0 +1,13 @@
+package au.com.codeka.carrot;
+
+/**
+ * The Bindings of a template when rendered.
+ *
+ * @author marten
+ */
+public interface Bindings
+{
```

Comment:
I know it might seem pointless, but I like to keep the code style the same throughout. Would you mind making sure all the { and on the previous line? "public interface Bindings {"